docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -160,7 +163,7 @@ The output looks similar to the following:
       "repository": "my_backup",
       "uuid": "XuBo4l4ISYiVg0nYUen9zg",
       "state": "SUCCESS",
-      "include_global_state": true,
+      "include_global_state": false,
```

Comment:
I think `true` is ok here, this is in an API response.

Reply:
I reverted this with a07e2a06fe4d9f64f56467bd3721b31de1ad6800.